standard/info/wikis/warcraft/info.lua

```diff
@@ -56,4 +56,5 @@ return {
 	defaultTeamLogoDark = 'Warcraft III logo.png', ---@deprecated
 	opponentLibrary = 'Opponent/Custom',
 	opponentDisplayLibrary = 'OpponentDisplay/Custom',
+	match2 = 1,
```

Comment:
somewhat `2` already
50 pages with 2v2 matches left, but those did never store into match to begin with
so from a data perspective everything available in match is available in match2